docs/source/running-autometa.rst

```diff
@@ -2,85 +2,266 @@
+#. Accept clusters that are estimated to be over 20% complete and 90% pure based on single-copy marker genes. These are default papameteres and can be altered to suit your needs.
+#. Unclustered contigs leftover will be re-clustered until no more acceptable clusters are yielded
+
+If you include a taxonomy table in the, Autometa will attempt to further partition the data based on ascending taxonomic specificity (i.e. in the order phylum, class, order, family, genus, species) when clustering unclustered contigs from a previous attempt. We found that this is mainly useful if you have a highly complex metagenome (lots of species), or you have several related species at similar coverage level.
+
+Use the following command to run the binning:
+
+.. code-block:: bash
+
+    # Archaeal binning
+    autometa-binning <path/to/kmers_norm.tsv> \
+    <path/to/coverage_table.tsv> <path/to/archaea.markers.tsv> \
+    <path/to/archaea_binning.tsv> --embedded-kmers <path/to/embedded_kmers.tsv> \
+    --taxonomy <path/to/taxonomy.tsv> --clustering-method <dbscan or hdbscan> --domain archaea
+
+    # Bacterial binning
+    autometa-binning <path/to/kmers_norm.tsv> \
+    <path/to/coverage_table.tsv> <path/to/bacterial.markers.tsv> \
+    <path/to/bacteria_binning.tsv> --embedded-kmers <path/to/embedded_kmers.tsv> \
+    --taxonomy <path/to/taxonomy.tsv> --clustering-method <dbscan or hdbscan> --domain bacteria
+
+The above command would generate the following files:
+
+- Final binning of each contig into a genome called ``<path/to/archaea_binning.tsv>`` and ``<path/to/bacteria_binning.tsv>`` for archaea and bacteria respectively
+
+You can view the complete command line opions using ``autometa-binning -h``
+
+7. Unclustered recruitment (Optional)
+-------------------------------------
+
+Supervised machine learning is used to classify the unclustered contigs to the bins that we have produced. This steop is optional and the results should be verified (see Note below) before going ahead with it.
+
+.. note::
+    The machine learning step has been seen to pick up contigs that not necessary belong to the genome. Careful inscpection of coverage and taxonomy should be done before you go ahead and use results from this step.
+
+Use the following command to run the unclustered recruitment step:
+
+.. code-block:: bash
+
+    # Archaea
+    autometa-unclustered-recruitment <path/to/kmers_norm.tsv> \
+    <path/to/coverage_table.tsv> <path/to/archaea_binning.tsv> \
+    <path/to/archaea.markers.tsv> <path/to/arachaea_unclustered_recruitment.tsv> \
+    --taxonomy <path/to/taxonomy.tsv> --classifier decision_tree
+
+    # Bacteria
+    autometa-unclustered-recruitment <path/to/kmers_norm.tsv> \
+    <path/to/coverage_table.tsv> <path/to/bacteria_binning.tsv> \
+    <path/to/bacteria.markers.tsv> <path/to/bacteria_unclustered_recruitment.tsv> \
+    --taxonomy <path/to/taxonomy.tsv> --classifier decision_tree
```

Comment:
I didn't even get to trying this - I just didn't have much hope after the last five or so commands failed.